bindings/src/test/java/io/pravega/storage/extendeds3/S3ProxyImpl.java

```diff
@@ -37,7 +37,7 @@
 import lombok.val;
 import org.apache.commons.httpclient.HttpStatus;
 import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
Unrelated.

Reply:
This was necessary, as it was depending on a transitive dependency coming via Hadoop client. Once I upgraded Hadoop client, the code compilation failed here. So, I used the replacement that was being brought in.